doc/nrf/protocols/matter/index.rst

```diff
@@ -22,7 +22,7 @@ The |NCS| allows you to develop applications with different versions of Matter,
 +--------------------------+-----------------------------------------------------+------------------------+
 | nRF Connect SDK version  | Matter specification version                        | `Matter SDK version`_  |
 +==========================+=====================================================+========================+
-| v2.3.99 (latest)         | :ref:`1.1.0 <ug_matter_overview_dev_model_support>` | 1.1.0.1                |
+| v2.4.0                   | :ref:`1.1.0 <ug_matter_overview_dev_model_support>` | 1.1.0.1                |
```

Comment:
@LuDuda, as discussed, suggesting this shortcut here. This will be updated to 2.4.99 after the release.
```suggestion
| | release|               | :ref:`1.1.0 <ug_matter_overview_dev_model_support>` | 1.1.0.1                |
```